.lingo

```diff
@@ -0,0 +1,11 @@
+lexicons:
+- codelingo/php
+tenets:
+- name: sql-concats
+  doc: Find all variable name ending in SQL that contain string concatenations.
+  comment: SQL query strings shouldn't be concatenated, these are susceptible to SQL injection.
+  match: |
```

Comment:
Mock comment

Reply:
@codelingobot capture this is the rule title

This is the rule body.

Reply:
@codelingobot capture